sdks/python/container/piputil.go

```diff
@@ -52,6 +53,18 @@ func pipInstallRequirements(files []string, dir, name string) error {
 	return nil
 }
 
+// isPackageInstalled checks if the given package is installed in the
+// environment.
+func isPackageInstalled(pkgName string) (bool, error) {
+	cmd := exec.Command("python", "-m", "pip", "show", pkgName)
+	if err := cmd.Run(); err != nil {
```

Comment:
does non-zero exit code have `err != nil` ?

Reply:
here, the Run method will return an exec.ExitError value so yes